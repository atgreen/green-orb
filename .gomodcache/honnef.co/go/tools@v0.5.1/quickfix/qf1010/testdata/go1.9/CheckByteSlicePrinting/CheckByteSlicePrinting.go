package pkg

import "fmt"

func fn() {
	type ByteSlice []byte
	type Alias1 = []byte
	type Alias2 = ByteSlice
	type Alias3 = byte
	type Alias4 = []Alias3

	fmt.Print(Alias1{}) //@ diag(`could convert argument to string`)
	fmt.Print(Alias2{}) //@ diag(`could convert argument to string`)
	fmt.Print(Alias4{}) //@ diag(`could convert argument to string`)
	fmt.Print()
	fmt.Fprint(nil)
}
