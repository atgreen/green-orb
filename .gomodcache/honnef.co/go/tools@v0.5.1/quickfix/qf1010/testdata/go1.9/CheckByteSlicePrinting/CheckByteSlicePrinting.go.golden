package pkg

import "fmt"

func fn() {
	type ByteSlice []byte
	type Alias1 = []byte
	type Alias2 = ByteSlice
	type Alias3 = byte
	type Alias4 = []Alias3

	fmt.Print(string(Alias1{})) //@ diag(`could convert argument to string`)
	fmt.Print(string(Alias2{})) //@ diag(`could convert argument to string`)
	fmt.Print(string(Alias4{})) //@ diag(`could convert argument to string`)
	fmt.Print()
	fmt.Fprint(nil)
}
